tests/test_file_utils.py

```
@@ -0,0 +1,1078 @@
+        """Test token estimation for nonexistent file"""
+        tokens = estimate_file_tokens("/nonexistent/file.txt")
+        assert tokens == 0
+
+    def test_check_files_size_limit_within_limit(self):
+        """Test file size check within limits"""
+        with tempfile.NamedTemporaryFile(mode="w", delete=False) as f:
+            f.write("small content")
+            f.flush()
+
+            within_limit, total_tokens, file_count = check_files_size_limit([f.name], max_tokens=10000)
+            assert within_limit == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if within_limit:` for truth checks

Replace with `within_limit`

(E712)